src/css/Parser.js

```diff
@@ -1986,6 +1986,11 @@ Parser.prototype = function() {
                     functionText = tokenStream.token().value;
                     this._readWhitespace();
                     expr = this._expr(true);
+                    if (expr === null) {
```

Comment:
@nschonni , do you have a preference on how to proceed? The list of dependent packages of parserlib is at https://www.npmjs.com/browse/depended/parserlib , so that's not too many.